sdk/src/main/java/com/meniga/sdk/models/transactions/MenigaTransaction.java

```diff
@@ -397,6 +404,17 @@ public Long getAccountId() {
 		return accountId;
 	}
 
+	/**
+	 * @return The account this transaction belongs to.
+	 */
+	public MenigaAccount getAccount() {
+		return account;
+	}
+
+	public void setAccount(MenigaAccount account) {
+		this.account = account;
```

Comment:
The convetion in the ask is to only have setters for fields that can be updated in the back end - such as category id etc.

Reply:
You’re right about those setters - I’ll try to refactor it one way or another so they’re not exposed.